lib/__tests__/__snapshots__/cli.test.mjs.snap

```diff
@@ -51,9 +51,14 @@ exports[`CLI --help 1`] = `
       A pattern of files to ignore (in addition to those in ".stylelintignore").
       Multiple can be set.
 
-    --fix
+    --fix [<mode>]
```

Comment:
[question] Should we support `--no-fix` also, as well as `--no-cache`, `--no-color`, `--no-validate`?

Reply:
meow does support it by default, the thing is we only document `--no` variant when the `false` is more relevant.
i.e. when the default is `true`
e.g. [`--no-validate`](https://stylelint.io/user-guide/cli/#--validate---no-validate)

Weirdly we have done it for [--no-color](https://stylelint.io/user-guide/cli/#--color---no-color) too.

Reply:
Thanks, I confirmed `--no-fix` sets `{ fix: false }` by default. But it seems a bit weird because `fix` is defined as `string`.

Also, I can't find a use case of `--no-fix`, so I think documenting `--no-fix` is unnecessary for now. Sorry for confusing.

Reply:
> But it seems a bit weird because `fix` is defined as `string`.

That's one of the reason why [I preferred to keep it at as `"boolean"`](https://github.com/stylelint/stylelint/pull/8106#discussion_r1827281601).
Since it's not documented and the default is `false`, it cannot be considered a regression if `--no-fix` stops working or throws.

Reply:
Similar to <https://github.com/stylelint/stylelint/pull/8106#discussion_r1836742056>, I don't think we need to support `--no-fix`. Even if it may work accidentally, it's undocumented. Considering undocumented behaviors makes things too complex.

Reply:
Ill resolve this thread.

---

off topic

I have a branch which uses `parseArgs` of `node:utils` instead of `meow`.
The complexity wasn't worth it, performance wise, so I didn't send a PR.

Now that you are saying that we don't have to support `--no-` prefix for all boolean flags, Ill come back to it.
Without all the logic necessary for `-no--` handling, the complexity burden is out of the window and the perf might be worth the refactor.

Thanks for the information.